app/assets/stylesheets/media-box.scss

```diff
@@ -1,4 +1,20 @@
-.media { margin: 10px; }
+.media {
+  margin: 10px;
+  position: relative;
+
+  .gallery-picture img { margin-bottom: 20px; }
+
+  .card-footer .footer-container {  font-size: 13px; }
+
+  .card-footer span { float: right; }
```

Comment:
We use those Bootstrap media objects for different kinds of elements:
```
$ grep -Rn "\"media" app/assets/templates
app/assets/templates/comment-stream_tpl.jst.hbs:3:    <div class="media">
app/assets/templates/contact_tpl.jst.hbs:6:  <div class="media-object pull-left">
app/assets/templates/contact_tpl.jst.hbs:10:  <div class="media-body">
app/assets/templates/likes-info_tpl.jst.hbs:3:    <div class="media">
app/assets/templates/reshares-info_tpl.jst.hbs:3:    <div class="media">
app/assets/templates/stream-element_tpl.jst.hbs:1:<div class="media {{#if showPost}} {{#if nsfw}} shield-off {{/if}} {{else}} shield-active {{/if}}">
app/assets/templates/reshare_tpl.jst.hbs:2:  <div class="media">
app/assets/templates/photo_tpl.jst.hbs:1:<div class="media">
```
(the output of `grep -Rn "\.media" app/views` is even longer)

and as far as I can tell you have to add the margin to have enough space for the card-footer.